### PR TITLE
Sort process list for consistent results

### PR DESCRIPTION
**Pull request by rehandalal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
+# Therapist
+Work out your commitment issues. A smart pre-commit hook for git.
```
Comment (willkg) [inline README.md]: This would be good on a t-shirt.

File: therapist/cli.py [hunk 2]
```diff
@@ -236,6 +236,7 @@ def run(**kwargs):
             output(UNSTAGED_CHANGES_MSG, end='\n\n')
 
         processes = list(runner.actions) + list(runner.plugins)
+        processes.sort(key=lambda x: x.name)  # Sort the list of processes for consistent results
```
Comment (willkg) [inline therapist/cli.py]: +1 for having a stable therapist.

File: README.md (hunk 1 above)
Comment (rehandalal) [reply README.md]: Maybe I'll get some for the next all hands

Comment (willkg) [reply README.md]: It'd be great on stickers, too. Plus they're cheaper.

File: therapist/cli.py (hunk 2 above)
Comment (willkg) [reply therapist/cli.py]: I'm going to giggle about this for the rest of the day.

**Pull request merged**